Sources/FirebaseFirestore/Query+Swift.swift

```diff
@@ -68,69 +68,94 @@ extension Query {
     return ListenerRegistration(boxed, instance)
   }
 
-  /* TODO: Implement these remaining methods:
-
-  public func whereFilter(_ filter: Filter) -> Query {
+  private func firestoreValueOrFail(_ value: Any) -> firebase.firestore.FieldValue {
+    guard let value = FirestoreDataConverter.firestoreValue(field: value) else {
+      fatalError("Unexpected value type")
```

Comment:
would it be possible to include the type in this fatal error so if we do run into this we have a bit more information on what was attempted to be converted?

Reply:
hmm, i guess there should be a way to get the type from an `Any`.. hmm..

Reply:
Yeah, `type(of: value)` works great! Thanks for the suggestion :)